rerun_cpp/src/rerun.hpp

```diff
@@ -2,29 +2,13 @@
 #pragma once
 
 // Auto-generated:
-#include "archetypes.hpp"
-#include "components.hpp"
-#include "datatypes.hpp"
+#include "rerun/archetypes.hpp"
+#include "rerun/components.hpp"
+#include "rerun/datatypes.hpp"
 
 // Manually written:
-#include "recording_stream.hpp"
+#include "rerun/recording_stream.hpp"
+#include "rerun/sdk_info.hpp"
 
-namespace rr {
-    /// The Rerun C++ SDK version as a human-readable string.
-    const char* version_string();
-} // namespace rr
-
-// ----------------------------------------------------------------------------
-// Arrow integration
-
-#include <arrow/api.h>
-
-namespace rr {
-    /// Encode the given arrow table in the Arrow IPC encapsulated message format.
-    ///
-    /// * <https://arrow.apache.org/docs/format/Columnar.html#format-ipc>
-    /// * <https://wesm.github.io/arrow-site-test/format/IPC.html#encapsulated-message-format>
-    arrow::Result<std::shared_ptr<arrow::Buffer>> ipc_from_table(const arrow::Table& table);
-} // namespace rr
-
-// ----------------------------------------------------------------------------
+// Generally not needed.
+// #include "rerun/arrow.hpp"
```

Comment:
Why not just remove?

Reply:
found it nice to still list out everything in here. But yeah that's clutter. off it goes!